core/runtime/src/state_type.rs

```diff
@@ -236,4 +237,35 @@ impl TryFrom<Vec<u8>> for Approved {
         let mut buf = s;
         Approved::from_bytes(&mut buf)
     }
+}
+
+#[derive(Encode, Decode, Clone, Debug, Default, PartialEq, PartialOrd, Eq, Ord, Hash)]
+pub struct Text(Vec<u8>);
```

Comment:
why the name is `Text`, the type is `Vec<u8>`?

Reply:
`Bytes` is suitable for use `Vec<u8>`, so merge `Text` to it.
12019f4cb2e34fe3de4097f0826fd6a6c02f7fb0